sort/sort.go

```diff
@@ -25,7 +25,13 @@ func MultithreadedSortComparators(comparators Comparators) Comparators {
 	copy(toBeSorted, comparators)
 
 	var wg sync.WaitGroup
-	chunks := chunk(toBeSorted, int64(runtime.NumCPU()))
+
+	numCPU := int64(runtime.NumCPU())
+	if numCPU%2 == 1 { // single core machine
+		numCPU++
+	}
```

Comment:
We're just checking to see if there are an odd number of CPUs.  In most cases a user will have 1, 2 and then some multiple of 2 thereafter.  If someone intentionally disabled a CPU to get an odd number (someone who has more than one core) this will also even that out.